pages.de/common/dotnet.md

```diff
@@ -0,0 +1,20 @@
+# dotnet
+
+> Plattformübergreifende Kommandozeilenandwendungen für .NET Core.
+> Mehr Informationen: <https://docs.microsoft.com/de-de/dotnet/core/tools/>.
+
+- Ein neues .NET Projekt initialisieren:
+
+`dotnet new {{template_name}}`
```

Comment:
You can translate the `template_name` here to German.

Reply:
"name" in german is "name" and I would consider "template" to be common knowledge.

Reply:
@gutjuri aparently had the same train of thought as me